frontend/dfx_react_typescript/README.md

```diff
@@ -0,0 +1,43 @@
+
+To get started, you might want to explore the project directory structure and the default configuration file. Working with this project in your development environment will not affect any production deployment or identity tokens.
+
+To learn more before you start working with dfx_react_typescript, see the following documentation available online:
+
+- [Quick Start](https://sdk.dfinity.org/developers-guide/quickstart.html)
+- [Developer's Guide](https://sdk.dfinity.org/developers-guide)
+- [Language Reference](https://sdk.dfinity.org/language-guide)
+
+## Important Note
+
+Before you run `dfx build`, you must update the `@dfinity/agent` property in the package.json
+
+to get the locataion of your `@dfinity/agent`, run `echo $HOME/.cache/dfinity/versions/<your-dfx-version>/js-user-library`. that should print something like this:
```

Comment:
Also, `echo $(dfx cache show)/js-user-library` will do the same.

Reply:
This is a workaround because anyone doing `dfx new` hardcodes the path, and if anyone cloned this project they'd have to update it (unless I'm missing something).  

Since it'll be in the `@dfinity/agent` package, I vote waiting on merging these PRs until then.